src/checklist/Checklist.tsx

```diff
@@ -35,6 +36,13 @@ const LoopTagWrapper = styled.div`
         margin: 0;
         margin-bottom: 4px;
     }
+    & button {
```

Comment:
Remove this and use the same button style that we use for "Show more details", defined in CheckItem.tsx on line 36. Now that this button layout will be used in multiple places, please create a new button - "ShowMoreButton.tsx" in the components > buttons folder. This generic button should accept a React.Dispatch<React.SetState<boolean>> function as a prop, and toggle the value. 

Replace your button and the button on CheckItem.tsx line 168 with this new button.